bin/phase-helpers.sh

```diff
@@ -872,122 +878,92 @@ __eapi6_src_install() {
 	einstalldocs
 }
 
-# @FUNCTION: has_version
-# @USAGE: [--host-root] <DEPEND ATOM>
-# @DESCRIPTION:
-# Return true if given package is installed. Otherwise return false.
-# Callers may override the ROOT variable in order to match packages from an
-# alternative ROOT.
-has_version() {
-
-	local atom eroot host_root=false root=${ROOT}
-	if [[ $1 == --host-root ]] ; then
-		host_root=true
-		shift
-	fi
+__best_version_and_has_version_common() {
```

Comment:
It's best to use the tripple underscore namespace for this function, which is automatically filtered from the saved environment since 29f3837382babcac38d4c0e6498620b0dbc6df67.